docs/source/changelog.rst

```diff
@@ -14,6 +14,7 @@ Changelog
         * Hyperlinked ComposeML (:pr:`653`)
         * Added FAQ (:pr:`620`)
     * Testing Changes
+        * Add trigger test for PP (:pr:`680`)
```

Comment:
we can just write out premium primitives

Reply:
Fixed.